src/modules/getusers.js

```diff
@@ -0,0 +1,13 @@
+export default async function getAllUsers(url) {
+  try {
+    const response = await fetch(url);
+
+    if (!response.ok) {
+      throw new Error(`Error! status: ${response.status}`);
+    }
+    const users = await response.json();
+    return users.result;
+  } catch (error) {
+    return `Error: ${error}`;
+  }
+}
```

Comment:
- [ ] Please change this function to the arrow function. You can check this. [why you should use the arrow function.](https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/)